vorto-dashboard/config.json

```diff
@@ -0,0 +1,8 @@
+{
+  "bosch_iot_suite": {
+    "client_id": "7d73ac34-d054-42c4-ae03-11095fea19d9",
+    "client_secret": "0E116C46B94D05651B181D4578B64500",
```

Comment:
Hi @timgrossmann 
Can you remove the credentials client-id and secret

Reply:
Sorry, will do that.
Thank you